### DISCUSSION TITLE
Welcome to Teams-Auto-Joiner Discussions!

### DISCUSSION THREAD

**Discussion by TobiasPankner** [TobiasPankner/Teams-Auto-Joiner#58] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tee123code** [comment]
Hi Tobias, your resources on teams auto joiner are really resourceful but how can I open through the desktop app and not chrome in the json file. I would like to know how I can switch between chrome and using the desktop app as well. What line of code do I need to replace, please help with this!

**Reply by Tee123code** [reply]
@TobiasPankner 
Hi Tobias, your resources on teams auto joiner are really resourceful but how can I open through the desktop app and not chrome in the json file. I would like to know how I can switch between chrome and using the desktop app as well. What line of code do I need to replace, please help with this!

**Comment by TobiasPankner** [comment]
> Hi Tobias, your resources on teams auto joiner are really resourceful but how can I open through the desktop app and not chrome in the json file. I would like to know how I can switch between chrome and using the desktop app as well. What line of code do I need to replace, please help with this!

This is not possible. The program is based on web automation.

**Reply by Tee123code** [reply]
Alright, thanks for your response! @TobiasPankner
